package-lock.json

```diff
@@ -1,8 +1,18248 @@
 {
   "name": "root",
   "version": "0.14.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
Have we officially moved to NPM 7 now?

Reply:
No, that wasn't an intentional decision.  I'll regenerate the lockfile and post a PR in a moment.  Thanks for letting me know.

Reply:
PR #6473

Reply:
Thank you for the quick reply and fix!